examples/walking_issues.php

```diff
@@ -6,9 +6,9 @@
  * Jira_Issues_Walker implicitly paging search request.
  * you don't need to care about paging request
  *
- * push(string $jql, string $navigable)
+ * push(string $jql, string $navigable = null, array $expanded = [])
```

Comment:
NOTE: Applies to other places (including non-comments as well).

Replace `[]` with `array()`.

---

Library is PHP 5.3 compatible and short array syntax was added in PHP 5.4 only.